docs/source/private-data/private-data.md

```diff
@@ -151,6 +151,15 @@ documentation on [transaction flow](../txflow.html).
    their copy of the private state database and private writeset storage. The
    private data is then deleted from the `transient data store`.
 
+The [Fabric Gateway](../gateway.html) takes care of collecting the required set of endorsements
```

Comment:
takes care of collecting >> collects

Reply:
The Fabric Gateway >> Fabric Gateway  ... (no "The" ... or alternatively, "The Fabric Gateway service" ...